File: assignment1/program1.swift
import Foundation

func main(){
    Task1()
}

func Task1(){
    print("Enter name: ")
    if let name = readLine(){
        var strArray = Array(name)
        var l = 0
        var r = name.count - 1;
        while(l < r){
            let tmp = strArray[l]
            strArray[l] = strArray[r]
            strArray[r] = tmp
            l += 1
            r -= 1
        }
        let strReversed = String(strArray)
        print("Reversed name is: \(strReversed)")
    }
}

main()

File: assignment1/program1.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Task1();
    }

    static void Task1()
    {
        Console.WriteLine("Enter your name: ");
        string str = Console.ReadLine();
        char[] strArray = str.ToCharArray();
        int l = 0; int r = strArray.Length - 1;
        while (l < r)
        {
            char tmp = strArray[l];
            strArray[l] = strArray[r];
            strArray[r] = tmp;
            l++; r--;
        }
        str = new string(strArray);
        Console.WriteLine("Reverse name is: " + str);
    }
    
}